components/webdriver_server/lib.rs

```diff
@@ -766,6 +766,42 @@ impl Handler {
         }
     }
 
+    // https://w3c.github.io/webdriver/#find-element-from-element
+    fn handle_find_element_element(
```

Comment:
How about let's use the name `element from element` for this feature?
`ElementElement` looks not easy to understand for me 😂

ex. `handle_find_element_from_element`, `FindElementFromElement`

Reply:
`element from element` also sounds better to me, but I wanted to keep the name consistent with that of the `webdriver` crate command: https://searchfox.org/mozilla-central/source/testing/webdriver/src/command.rs